Changelog.md

```diff
@@ -2,6 +2,9 @@
 
 <!-- - title: description ([#](https://github.com/fossas/fossa-cli/pull/#)) -->
 
+## v3.7.4
+- Gradle: Fix possible ConcurrentModificationException that can ocurr when getting dependencies ([#1166](https://github.com/fossas/fossa-cli/pull/1166))
```

Comment:
Nit: spelling

```suggestion
- Gradle: Fix possible ConcurrentModificationException that can occur when getting dependencies ([#1166](https://github.com/fossas/fossa-cli/pull/1166))
```

Reply:
Also, should we change the URL to point at this PR? I think we'll end up closing #1166, so it might be confusing to point at it

Reply:
I left it this way because #1166 has the most context, but I guess it's easy enough for someone to find that PR via the description in this one.